modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -1968,8 +1968,8 @@ If you want to decimate the image by factor of 2 in each direction, you can call
     // specify fx and fy and let the function compute the destination image size.
     resize(src, dst, Size(), 0.5, 0.5, interpolation);
 @endcode
-To shrink an image, it will generally look best with CV_INTER_AREA interpolation, whereas to
-enlarge an image, it will generally look best with CV_INTER_CUBIC (slow) or CV_INTER_LINEAR
+To shrink an image, it will generally look best with INTER_AREA interpolation, whereas to
+enlarge an image, it will generally look best with INTER_CUBIC (slow) or INTER_LINEAR
```

Comment:
Why didn't you add `cv::` prefix to enum values ( `INTER_LINEAR ==> cv::INTER_LINEAR`)? This modification would allow users jump to corresponding documentation.